watch-library/hardware/watch/watch_private.c

```diff
@@ -255,33 +255,22 @@ int _write(int file, char *ptr, int len) {
     return 0;
 }
 
-static char buf[256] = {0};
-
 int _read(int file, char *ptr, int len) {
     (void)file;
-    int actual_length = strlen(buf);
-    if (actual_length) {
-        memcpy(ptr, buf, min(len, actual_length));
-        return actual_length;
+
```

Comment:
This avoids the following issues:

- calling read more than once before cdc_task happened would give us the same data
- not calling read before cdc_task happened would silently drop data
- inability to read '\0'